### PR TITLE
Update ui.rst

### PR DESCRIPTION
**Pull request by AnthonyLim23**
Cleaned up documentation.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by samueljackson92** [comment]
@AnthonyLim23 Line 79 also seems to have an additional space which is causing that bullet point to show up weird in the markdown file preview.

**Comment by AnthonyLim23** [comment]
Line 79 was blank, so I removed it. Does that show up better? How do I view the rendered text (so I can check myself in future)?

File: docs/source/release/v3.9.0/ui.rst [hunk 1]
```diff
@@ -76,7 +76,6 @@ SliceViewer Improvements
 ------------------------
 
  - Added ability to view data using non orthogonal axes. View can be toggled on or off. When non orthogonal view is toggled, peak viewer and line vewer tools are disabled.
```
Comment (samueljackson92) [inline docs/source/release/v3.9.0/ui.rst]: Sorry I meant 78. This line here beginning "Added ability to view data using..."

**Comment by samueljackson92** [comment]
@AnthonyLim23 Oh and I forgot to say I just went to the commit diff (File changed at the top) and then clicked "View" on `ui.rst`. Then github shows a preview of the rst file.

**Comment by AnthonyLim23** [comment]
thats great. Thank you

**Comment by AnthonyLim23** [comment]
is there anything else you think I need to do?

**Comment by samueljackson92** [comment]
Changes are only to documentation and it looks good to me. :shipit:

**Review by samueljackson92 — APPROVED** [review]

**Pull request merged**